<launch>
  <param name="/use_sim_time" value="true"/>

  <arg name="use_static_tf" default="false"/>
 
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find leo_driving)/rviz/view_point.rviz"/>

  <node pkg="leo_driving" type="leo_driving_node" name="leo_driving_node" output="screen">
	<param name="Kpy_param" value="3.5"  /> 
	<param name="Kpy_param_rot" value="0.3" /> 
	<param name="linear_vel" value="0.05" /> 
	
	<param name="global_angle_boundary" value="0.05" /> 
	<param name="global_dist_boundary" value="0.3" /> 
	<param name="adjusting_y_err_bound" value="0.005" /> 
	
  </node>

<node if="$(arg use_static_tf)" pkg="tf" type="static_transform_publisher" name="base_link_to_rp" args="0 0 0 3.141592 0 0 base_link laser 100" />
<node if="$(arg use_static_tf)" pkg="tf" type="static_transform_publisher" name="base_link_to_velo" args="0 0 0 0 0 0 base_link velodyne 100" />
<node if="$(arg use_static_tf)" pkg="tf" type="static_transform_publisher" name="base_link_to_imu" args="0 0 0 0.0 0 0 base_link imu 100" />

</launch>

